09_timer/timer.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+class Timer
+  @timer = Timer.new
+  attr_accessor :seconds
+  def initialize(seconds = 0)
+    @seconds = seconds
+  end
+
+  def time_string
+    seconds = @seconds % 60
+    minutes = (@seconds / 60) % 60
+    hours = @seconds / (60**2)
+    "#{padded(hours)}:#{padded(minutes)}:#{padded(seconds)}"
+  end
+
+  def padded(num)
```

Comment:
Method names should be verbs. In this case it would be `pad` or `left_pad` or [rjust](https://apidock.com/ruby/String/rjust)

Reply:
Still pending^